docs/docs/behaviors/macros.md

```diff
@@ -198,12 +222,35 @@ To achieve this, a combination of a 0ms wait time and splitting the press and re
 
 #### Layer + Modifier
 
-```
-wait-ms = <0>;
-bindings
-    = <&macro_press &mo 1 &kp LSHFT>
-    , <&macro_pause_for_release>
-    , <&macro_release &mo 1 &kp LSHFT>;
+```dts
+/**
+ * Temporarily switches to a layer (`&mo`) while a modifier is held.
+ * Analogous to QMK's `LM()`, using a parameterized macro.
+ *
+ * Params:
+ *  1. Layer to switch to
+ *  2. Modifier to press while layer is active
+ *
+ * Example:
+ *  `&lm NUM_LAYER LSHIFT`
+ */
+lm: lm {
+    label = "LAYER_MOD";
+    compatible = "zmk,behavior-macro-two-param";
+    wait-ms = <0>;
+    tap-ms = <0>;
+    #binding-cells = <2>;
+    bindings
+        = <&macro_param_1to1>,
+        <&macro_press &mo MACRO_PLACEHOLDER>,
+        <&macro_param_2to1>,
+        <&macro_press &kp MACRO_PLACEHOLDER>,
+        <&macro_pause_for_release>,
+        <&macro_param_2to1>,
+        <&macro_release &kp MACRO_PLACEHOLDER>,
+        <&macro_param_1to1>,
+        <&macro_release &mo MACRO_PLACEHOLDER>;
```

Comment:
```suggestion
        = <&macro_param_1to1>
        , <&macro_press &mo MACRO_PLACEHOLDER>
        , <&macro_param_2to1>
        , <&macro_press &kp MACRO_PLACEHOLDER>
        , <&macro_pause_for_release>
        , <&macro_param_2to1>
        , <&macro_release &kp MACRO_PLACEHOLDER>
        , <&macro_param_1to1>
        , <&macro_release &mo MACRO_PLACEHOLDER>
        ;
```
Style consistency with the rest of the macro docs

Reply:
Done